.github/workflows/ci.yml

```diff
@@ -277,14 +280,14 @@ jobs:
         DOCKER_TOKEN: ${{ secrets.K8SSANDRA_DOCKER_HUB_PASSWORD }}
       run: |
         docker login -u $DOCKER_USERNAME -p $DOCKER_TOKEN
-        echo "Publishing release $(git rev-parse --short HEAD) in Docker Hub"
+        echo "Publishing release $(git rev-parse --short HEAD)${{ matrix.storage-provider }} in Docker Hub"
         python setup.py build
         # Push Docker image tagged with the short commit sha
-        docker build -t k8ssandra/medusa:$(git rev-parse --short HEAD) -f k8s/Dockerfile .
-        docker push k8ssandra/medusa:$(git rev-parse --short HEAD)
+        docker build -t k8ssandra/medusa:$(git rev-parse --short HEAD)${{ matrix.storage-provider }} -f k8s/Dockerfile${{ matrix.storage-provider }} .
+        docker push k8ssandra/medusa:$(git rev-parse --short HEAD)${{ matrix.storage-provider }}
         # Push Docker image tagged as "master"
-        docker tag k8ssandra/medusa:$(git rev-parse --short HEAD) k8ssandra/medusa:master
-        docker push k8ssandra/medusa:master
+        docker tag k8ssandra/medusa:$(git rev-parse --short HEAD)${{ matrix.storage-provider }} k8ssandra/medusa:master${{ matrix.storage-provider }}
+        docker push k8ssandra/medusa:master${{ matrix.storage-provider }}
```

Comment:
``:latest`` is more of a common strategy for containers and there's a special handling for them in Kubernetes also. Is there a reason to keep the ``:master`` tag-name instead?

Reply:
Isn't `:latest` supposed to correspond to the latest release? It's not the case here, we're publishing beta images out of the `master` branch.